### PR TITLE
feat(enum): out put enum json

### PR DESCRIPTION
**Pull request by nameliuqi**

### CODE REVIEW

**Comment by nameliuqi** [comment]
运行查看例子 `rm -rf ./test/thriftNew/out/ && yarn build && node ./bin/dts-from-thrift -p ./test/thriftNew/examples/ -o ./test/thriftNew/out/ --new --enum-json enum.json`

File: src/cmd.ts [hunk 1]
```diff
@@ -116,6 +119,12 @@ const tasks = thriftFiles.map(async filename => {
 const rTasks: Array<Promise<any>> = [];
 rTasks.push(
   Promise.all(tasks).then(entityList => {
+    if (options.enumJson) {
+      console.log('start hanle enum');
```
Comment (LPegasus) [inline src/cmd.ts]: hanle => handle?

File: src/thriftNew/print.ts [hunk 2]
```diff
@@ -354,3 +354,27 @@ export function printComments(
...
+/**
+ * @description 获取includeMap中的enum值
+ */
+export function printEnumsObject(includeMap: {
+  [key: string]: RpcEntity;
+}): { [key: string]: number } {
+  const res: { [key: string]: number } = {};
+  Object.keys(includeMap).forEach(key => {
+    const rtn = includeMap[key];
+    console.log('hanle rtn', rtn.fileName);
```
Comment (LPegasus) [inline src/thriftNew/print.ts]: hanle => handle?

**Review by LPegasus — COMMENTED** [review]
LGTM

**Pull request merged**